config/locales/ja.yml

```diff
@@ -1,3 +1,7 @@
 ja:
   logged_out: 'ログインに成功しました'
   could_not_log_in: 'Shopifyストアにログインできませんでした'
+  enable_cookies_heading: "Enable cookies from %{app}"
+  enable_cookies_body: "You must manually enable cookies in this browser in order to use %{app} within Shopify."
+  enable_cookies_footer: 'Cookies let the app authenticate you by temporarily storing your preferences and personal information. They expire after 30 days.'
+  enable_cookies_action: 'Enable cookies'
```

Comment:
These all are in English (I guess you know that :) ), and you're missing a new line here.